contributing/README.md

```diff
@@ -34,6 +34,15 @@ Start the title with `[ComponentName]` to identify which component a change affe
 [FooBar] Removes the deprecated fooWithBar:(Bar*)bar method.
 ```
 
+### Pull request continuous integration for new contributors
```

Comment:
I would like to see CI run on all new PRs. I think it's valuable for contributors to get automated build/clang-format feedback before a team member looks at changes. It will also save us (those with write access) from having to manually comment on broken/unformatted code.

Reply:
Please file a feature request with kokoro for this.

Reply:
I can't find "kokoro" under the Google organization on GitHub nor in a google search. Can you please provide a link to their bug tracker?

Reply:
[go/kokoro-bug](http://go/kokoro-bug)

Reply:
I don't think we should be documenting this as a feature/process. I think it's better to list it as an outstanding bug in the tooling.

Reply:
Which aspect of this PR are you referring to?

Reply:
This new addition is part of a "General Contributing Guidelines". If we consider kokoro's behavior to be a defect/bug, then this should probably be listed under and described as a caveat/faq/mitigation rather than as a guideline.

Reply:
*nods* that's fair. At least for now though this is the behavior that we have to work within. I'll list the bug you filed here so that we can follow-up.